api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentGroupController.java

```diff
@@ -332,6 +361,180 @@ private Payment getPayment(PaymentFeeLink paymentFeeLink, String paymentReferenc
             .orElseThrow(() -> new PaymentNotFoundException("Payment with reference " + paymentReference + " does not exists."));
     }
 
+    @ApiOperation(value = "Record a Bulk Scan Payment", notes = "Record a Bulk Scan Payment")
+    @ApiResponses(value = {
+        @ApiResponse(code = 201, message = "Bulk Scan Payment created"),
+        @ApiResponse(code = 400, message = "Bulk Scan Payment creation failed"),
+        @ApiResponse(code = 422, message = "Invalid or missing attribute")
+    })
+    @PostMapping(value = "/payment-groups/{payment-group-reference}/bulk-scan-payments-strategic")
+    @ResponseBody
+    @Transactional
+    public ResponseEntity<PaymentDto> recordBulkScanPaymentStrategic(@PathVariable("payment-group-reference") String paymentGroupReference,
+                                                                     @Valid @RequestBody BulkScanPaymentRequestStrategic bulkScanPaymentRequestStrategic,
+                                                                     @RequestHeader(required = false) MultiValueMap<String, String> headers) throws CheckDigitException {
```

Comment:
Do we need this explicitly @RequestHeader? Because i couldn't see this in the existing end point.

Reply:
with these changes bulk scan api called via payhub and not paybubble , request header is related to that purpose

payhub (request headers) -> (request header) bulk scan api

we need to call two different endpoints using one request header and hence propagation is related to that

Reply:
I think what we are trying to achieve is to make a BSP Call from PayHub, so what is the reason we are intercepting Paybubble Headers & propagating the same to pass to BSP, we can set required headers for BSP in Payhub.